Makefile

```diff
@@ -146,6 +146,11 @@ gcp: kustomize helmify yq
 	cat infrastructure-components.yaml | $(HELMIFY) -generate-defaults -image-pull-secrets charts/cluster-api-provider-gcp
 	rm infrastructure-components.yaml tmp.yaml tmp2.yaml
 
+	# Delete the secret file since we are managing that ourselves
+	rm charts/cluster-api-provider-gcp/templates/manager-bootstrap-credentials.yaml
+	# Add the bootstrapMode toggle to easily nullify the credentials.
+	$(YQ) -i ".bootstrapMode=false" charts/cluster-api-provider-gcp/values.yaml
```

Comment:
Can you default this to `true` so it is the same as on AWS and possible others using the chart have the simple installation work. Then in our artifact we will set the default to `false` and pass `true` as a bootstrap flag.